roles/reproducer/tasks/validations.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Ensure we pass needed parameter
+  ansible.builtin.assert:
+    that:
+      - cifmw_use_libvirt is defined
+      - cifmw_use_libvirt | bool
+    msg: >-
+      Please ensure you pass "cifmw_use_libvirt: true" to the
+      deployment!
+    quiet: true
+
+- name: Verify ansible_host is defined in inventory
+  when: cifmw_target_host != 'localhost'
+  delegate_to: localhost
+  ansible.builtin.assert:
+    that: ansible_host != 'localhost'
+    msg: >-
+      Please ensure you define your ansible_host in your inventory file
+      https://ci-framework.readthedocs.io/en/latest/quickstart/03_deploy_infra.html#inventory-file
+    quiet: true
+
+- name: Verify ansible_connection is set to "local" in provided inventory
+  delegate_to: localhost
+  ansible.builtin.assert:
+    that: ansible_connection == 'local'
+    msg: >-
+      Please ensure you define your ansible_connection to local for localhost in your inventory file
+      https://ci-framework.readthedocs.io/en/latest/quickstart/03_deploy_infra.html#inventory-file
+    quiet: true
+
+- name: Verify we are running on supported Centos version
+  when: ansible_distribution == 'CentOS'
+  ansible.builtin.assert:
+    that:
+      - ansible_distribution_version >= '9'
+    msg: >-
+      Please ensure host: {{ cifmw_target_host }} is running a supported version of Centos
+      https://ci-framework.readthedocs.io/en/latest/quickstart/02_prepare_env.html#tested-environment
+    quiet: true
+
+- name: Verify we are running on supported RHEL version
+  when: ansible_distribution == 'RedHat'
+  ansible.builtin.assert:
+    that:
+      - ansible_distribution_version >= '9.2'
```

Comment:
@lewisdenny We have moved all the downstream CI jobs to RHEL-9.4. Can we bump the RHEL conditional to RHEL-9.4?

Reply:
Good idea, done!